### PR TITLE
Add react-native-orientation

### PR DESCRIPTION
**Pull request by retyui**

### CODE REVIEW

**Comment by retyui** [comment]
dead project https://github.com/yamill/react-native-orientation/pull/323

File: definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// @flow
```
Comment (villesau) [inline definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js]: Flow annotation is not needed in libdefs

File: definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { Orientations } from "react-native-orientation";
```
Comment (villesau) [inline definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js]: Importing outside lib defs, the resulting type is `any`. And that said, unfortunately the test suite does not yet support importing other than builtin libdefs inside module declaration :/

At the moment best bet would be to copy paste the Orientations types over, or then accepting that it's any

Comment (retyui) [reply definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js]: This is my IDE added this import) 
Removed :+1:

**Comment by villesau** [comment]
GJ!

**Comment by retyui** [comment]
@villesau
Thanks for helping to indicate the best practices)

**Pull request merged**